src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs

```diff
@@ -46,7 +46,7 @@ internal class DefaultHealthCheckService : HealthCheckService
             }
 
             var totalTime = ValueStopwatch.StartNew();
-            Log.HealthCheckProcessingBegin(_logger);
+            HealthCheckProcessingBegin();
```

Comment:
I don't like the change to not have these methods in a static `Log` type.

When I read `HealthCheckProcessingBegin();` nothing tells me this is for logging. When I read this I see it is preparing the service to start processing. Telling logging methods apart from regular framework logic is valuable.